.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "remote"]
 	path = remote
-	url = git@github.com:MusicConnectionMachine/api.git
+	url = https://github.com/MusicConnectionMachine/api.git
```

Comment:
@Henni at first yes, but [now](https://github.com/MusicConnectionMachine/RelationshipsG4/pull/39) it works via ssh